timesource/timesource.go

```diff
@@ -239,19 +203,29 @@ func (s *NTPTimeSource) runPeriodically(fn func() error) error {
 				}
```

Comment:
Thank you for your PR @osmaczko , I think the best way to solve the relate issue would be adjust logic `else if` here (e.g. add indicator that already tried first time but got error, so next invoking to `GetCurrentTime` should not wait and return time using latest offset), we should try our best to wait(up to 2 seconds as we have `DefaultRPCTimeout`) timesource to get the offset for the first time at least in some cases.

Reply:
> we should try our best to wait(up to 2 seconds as we have DefaultRPCTimeout) timesource to get the offset for the first time at least in some cases.

I agree, this is exactly what I do here: https://github.com/status-im/status-go/blob/52cd6bbb2c871d48f788c54600cfe2dfc7748fe4/timesource/timesource.go#L218-L226

Reply:
awesome, i didn't notice this change before my previous comment, LGTM! 👍